src/DFC_BusinessOntology.owl

```diff
@@ -1303,474 +1309,6 @@
     <Declaration>
         <Datatype abbreviatedIRI="xsd:duration"/>
     </Declaration>
-    <Declaration>
```

Comment:
We only just introduced these Declarations (with the Business Definitions), do we not need them ? 😕

Reply:
If we are removing them, it should be noted in CHANGELOG.md

Reply:
It is really strange because it looks like these declarations are coming from the facets taxonomies which is imported now so that's normal they don't belong to this file. However, I didn't change anything regarding this question when I worked on it so I don't know what I did, but this happened. I can update the changelog